### ISSUE TITLE
useless drills

### ISSUE THREAD

**Issue by theMegaTech** [GTNewHorizons/GT-New-Horizons-Modpack#11846]
### Your GTNH Discord Username

Мега.#3065

### Your Pack Version

2.2.3.0

### Your Proposal

GT Drills should have 3x3+ mining. Maybe MV - 3x3 and HV - 5x5 or something like that.

### Your Goal

cuz only option to mine more than single block at time - Tinkers' Constructs primitive hammers or magic (and afaik magic is intended to be optional). 

### Your Vision

I think we need electrical option to mine BEFORE multiblock miners, cuz they're making mining not with miner useless. And only option for that - gravisuite drill, that is post-platline. That feature exist even in GTCEU (GrehgTech Cringe Edition Universal), why don't backport it?

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by chochem** [comment]
> I think we need electrical option to mine BEFORE multiblock miners

so single block miners?

**Comment by theMegaTech** [comment]
> > I think we need electrical option to mine BEFORE multiblock miners
> 
> so singleblock miners?

They're painfully slow and are meaningless to use comparing to mining with hammer

**Comment by theMegaTech** [comment]
and actually many people like to mine by hand, why don't give them feeling of progression too?

**Comment by chochem** [comment]
> They're painfully slow and are meaningless to use comparing to mining with hammer

I mean independent of the rest of the discussion, this part is just blantantly wrong.

If you really prefer manual mining I would recommend to not dismiss magic like that. Some players mine with BM even in pretty late ages, its quite powerful.

**Comment by theMegaTech** [comment]
> If you really prefer manual mining I would recommend to not dismiss magic like that. Some players mine with BM even in pretty late ages, its quite powerful.
Yes, magic can solve mining issues, but isn't gtnh first of all gregtech-centered pack? I think more powerful drills will suit gtnh very well.

**Comment by boubou19** [comment]
>  but isn't gtnh first of all gregtech-centered pack?

GT:NH is centered around GT for the tech part, and centered around thaumcraft for the magical part.

**Comment by dvdmandt** [comment]
IMHO 5x5 mining is something that should be reserved for meme-tier equipment and around MV is when you start getting good hammers etc, so I don't feel like they are outdated at all at that point. Magic is intended to be optional, but give nice-to-have goodies, like for example an earlier option to nice mining tools.

I would however not really mind a good tunneling/space clearing option around mid-late HV, that could handle stone, dirt, gravel and similar, as well as the bane of my existence nether flesh, possibly voiding whatever it breaks to not make it an OP mining tool but rather a useful base building tool.

**Comment by theMegaTech** [comment]
5х5 is too much, maybe, agreed.
Hammers aren't outdated until gravisuit drill or funny ichor pick, but still it'd be better to have more electric tools, that doesn't need repairing every vein.
>that could handle stone, dirt, etc.
That's a reason to have drill too. One mining tool is much more convenient than fastbreaking shovel and hammer.

**Comment by Tybob12** [comment]
### Request to Reopen – Scalable AoE Drills

I’d like to request reopening this issue because the idea is still very relevant today. GregTech drills remain single-block tools, and there’s currently no electric option for AoE mining without using Tinkers’ Construct hammers, Botania’s Terra Shatterer, or other magical alternatives.

As mentioned in earlier comments, a large mining area, like 5×5 or greater, could be overpowered if introduced too early. However, even a basic 3×3 option at lower tiers would provide a balanced, tech-focused alternative for players who prefer rechargeable tools and want to avoid magic progression. Higher-tier drills, such as EV and IV tiers, could later expand to 5×5 or larger areas as a logical upgrade path, increasing both AoE size and internal battery capacity with tier progression.

This approach wouldn’t invalidate magical tools since GregTech drills can’t be enchanted with Fortune. That distinction keeps them focused on terraforming, tunneling, and large-scale construction, while magic remains better for ore-yield efficiency.
